.github/workflows/emscripten.yml

```diff
@@ -61,5 +60,9 @@ jobs:
         run: |
           pyodide venv .venv-pyodide
           source .venv-pyodide/bin/activate
-          npm install pyodide@${{ env.PYODIDE_VERSION }}
-          node tools/emscripten/pytest_scikit_image_pyodide.js --pyargs skimage
+          pip install dist/*.whl
+          pushd doc
```

Comment:
Why use `pushd` here and not `cd`? Why do we need to run the below in `doc/`?

Reply:
Sure, we can use `cd` as well, that shouldn't be an issue. We need to run the below code in `doc/` so that `pytest` correctly picks up the installed version of `skimage`, instead of trying to import `skimage` from the root directory of the repository (which won't be installed and therefore won't have the Emscripten-compiled object files).

Reply:
d789b32c0e59a6af5ee85e969db755fd4931583c added some more explanation that I hope will be useful for you and other maintainers who come to the workflow in the future.

Reply:
Thanks!